src/main/java/mate/academy/mybookshop/service/UserServiceImpl.java

```diff
@@ -23,6 +25,7 @@ public UserResponseDto registerUser(UserRegistrationRequestDto request)
         }
 
         UserEntity user = userMapper.toEntity(request);
+        user.setPassword(passwordEncoder.encode(request.getPassword()));
```

Comment:
we should also set the default `USER` role here

Reply:
added